llvm/lib/FuzzMutate/IRMutator.cpp

```diff
@@ -360,7 +360,11 @@ void InsertFunctionStrategy::mutate(BasicBlock &BB, RandomIRBuilder &IB) {
 
   auto RS = makeSampler(IB.Rand, Functions);
   Function *F = RS.getSelection();
-  if (!F) {
+  auto IsUnsupportedTy = [](Type *T) {
```

Comment:
Pls add a comment here to describe what problem are we solving. Probably a link to metadata type argument in the language reference.